chat/src/main/java/bisq/chat/Citation.java

```diff
@@ -60,12 +69,12 @@ public bisq.chat.protobuf.Citation toProto(boolean serializeForHash) {
 
     public static Citation fromProto(bisq.chat.protobuf.Citation proto) {
         return new Citation(proto.getAuthorUserProfileId(),
-                proto.getText(), proto.getChatMessageId());
+                proto.getText(), Optional.of(proto.getChatMessageId()));
```

Comment:
The protobuf field should be set optional as well and we should use hasChatMessageId to check its presence.